src/app/core/component/cardrenderer/card-renderer.component.html

```diff
@@ -1,26 +1,28 @@
 <div class="card-container">
-    <mat-card (click)="navigate(entity)"
-              *ngFor="let entity of uriEntities"
-              class="card box">
-        <mat-card-title style="margin-bottom: 1rem">{{entity.name}}</mat-card-title>
-        <mat-card-content>
-            <img mat-card-image *ngIf="entity.iconUrl else noimage" [src]="entity.iconUrl">
-            <ng-template #noimage><img mat-card-image src="//via.placeholder.com/350x150"></ng-template>
-        </mat-card-content>
-        <mat-card-footer *ngIf="showLinks">
+  <mat-card *ngFor="let entity of uriEntities"
+            class="card box">
+    <mat-card-title (click)="navigate(entity)" style="margin-bottom: 1rem">{{entity.name}}</mat-card-title>
+    <mat-card-content>
```

Comment:
I'm not sure if it's really intuitive to click on the title for the navigation to the pattern... maybe you can add a "Details" button?